src/main/java/org/broadinstitute/hellbender/engine/filters/BaseQualityReadFilter.java

```diff
@@ -0,0 +1,36 @@
+package org.broadinstitute.hellbender.engine.filters;
+
+import org.broadinstitute.hellbender.cmdline.Argument;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import java.io.Serializable;
+
+public final class BaseQualityReadFilter extends ReadFilter implements Serializable {
```

Comment:
I have removed this filter in favor of using BaseQualityReadTransformer to mask low-quality bases with 'N' and then applying AmbiguousBaseReadFilter to filter reads with too many 'N's - thus taking into account N's due to both low base quality as well as low-complexity subsequences.